libs/interfaces/src/lib/vaccines-report.ts

```diff
@@ -1,16 +1,12 @@
 import { CITIES_TYPE, HEALTH_CARE_SERVICES_TYPE } from '@vacgaps/constants';
 
-interface Time {
-  hour: number;
-  minutes: number;
-}
 export interface VaccinesReport {
   branchName: string;
   city: CITIES_TYPE[number];
   address: string;
   healthCareService: HEALTH_CARE_SERVICES_TYPE[number];
-  startTime?: Time;
-  endTime?: Time;
+  startTime?: string;
```

Comment:
Isn't there a type for DateTime?

Reply:
there is, but when we get the report from the backend, this field is a string. i don't know where it makes sense for me to parse this into a Date object, and it seems angular still accepts this as a Date.